data/yam/agama/auto/template.libsonnet

```diff
@@ -27,7 +28,7 @@ function(addon_ha_reg_code='',
     id: product,
     [if registration_code != '' then 'registrationCode']: registration_code,
   },
-  [if root == true then 'root']: base_lib['root'],
+  [if root || root_sshkey then 'root']: if root_sshkey then base_lib['root_sshkey'] else base_lib['root'],
```

Comment:
why do you need this change? in qemu is not needed and in remote workers should already be there.

Reply:
cause for s390x and ppc64le we need  'password and sshkey' to make ssh works.

Reply:
ok, when you address this https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21895#discussion_r2065380588 and use interactive installation in the import we already have done that, so doesn't need to be done here.

Reply:
I'll try puppeteer test_default_installation this afternoon,if it works then no need to change here.